cc/modules/protocol/mpc/snn/src/impl/division.cpp

```diff
@@ -25,7 +25,7 @@ int Div::funcDiv(const vector<mpc_t>& a, const vector<mpc_t>& b, vector<mpc_t>&
 }
 int Truediv::funcTruediv(
   const vector<mpc_t>& a, const vector<mpc_t>& b, vector<mpc_t>& c, size_t size) {
-  return GetMpcOpInner(DivisionV2)->Run(a, b, c, size);
+  return GetMpcOpInner(ReciprocalDiv)->Run(a, b, c, size);
```

Comment:
it will be better if we don't change the current implementation.

Reply:
I have recovered it and I will decide the future work to add a new implementation for reciprocal.